src/background/telemetry.js

```diff
@@ -2,121 +2,189 @@
+      record_on_release: true,
+    },
+    "itemDelete": {
+      methods: ["itemDelete"],
+      objects: ["manager", "doorhanger"], // depends on where the user initiated the action
+      extra_keys: [], // item GUID should be in the extra field. value is null.
+      record_on_release: true,
+    },
+    "itemSelected": {
+      methods: ["itemSelected"],
+      objects: ["manager", "doorhanger"], // fires when a user clicks on an item in the item list.
+      extra_keys: [],  // value is null
+      record_on_release: true,
+    },
+    "copyPassword": {
+      methods: ["copyPassword"], // fires when user copies password from an item
+      objects: ["itemDetailManager", "itemDetailDoorhanger"],
+      extra_keys: [], // item GUID should be in the extra field. value is null.
+      record_on_release: true,
+    },
+    "copyUsername": {
+      methods: ["copyUsername"], // fires when user copies username from an item
+      objects: ["itemDetailManager", "itemDetailDoorhanger"],
+      extra_keys: [], // item GUID should be in the extra field. value is null.
+      record_on_release: true,
+    },
+    "click": {
+      methods: ["click"],
+      objects: [
+        "faq", // click on the FAQ link
+        "giveFeedback", // click on the feedback UI
+        "toolbar", // click on the toolbar button TODO where do I do this? think it's been deleted...
+        "settingsMenu", // click on the avatar to open the settings menu
+        "accountSettings",  // settings menu: account settings option
+        "getMobile", // settings menu: 'connect a device' menu item
+        "signinSync", // settings menu: sign in to sync menu item (shown when logged out of FxA/Sync)
+        "sortMenu", // value: "lastUsed", or "lastChanged", or "name"
+      ],
+      extra_keys: [],
+      record_on_release: true,
+    },
+    "show": {
+      methods: ["show"],
+      objects: [
+        "newItem", // new item view, per-login.
+        "itemListManager", // list logins in manage page, I guess each page load?
+        "itemListDoorhanger", // list logins in doorhanger, I guess each time the doorhanger's shown.
+        "itemDetailManager", // view item details in manage UI, per-login.
+        "itemDetailDoorhanger", // view details in doorhanger, per-login.
+        "itemEdit", // item edit view, per-login.
+        "deleteConfirm", // delete confirm modal, per-login
+
+        // not implemented:
+
+        "connectAnotherDevice", // TODO: not sure this makes sense as a "show" event, unless it's for the unimplemented mobile upsell (#97?)
+      ],
+      extra_keys: [],
+      record_on_release: true,
+    },
+    "openWebsite": {
+      methods: ["openWebsite"], // fires when a user clicks to open the website inside a login
+      objects: ["itemDetailManager", "itemDetailDoorhanger"],
+      extra_keys: [], // item GUID should be in the extra field. value is null.
+      record_on_release: true,
+    },
+    // TODO: this one is still in progress
```

Comment:
we don't need to land all the telemetry in one PR (unless that's what you want).